src/coreclr/tests/src/Common/Directory.Build.targets

```diff
@@ -21,17 +21,34 @@
       <RunTimeDependencyExcludeFiles Include="@(RunTimeDependencyExclude -> '%(FileName)%(Extension)')" />
       <RunTimeDependencyExcludeFiles Include="@(RunTimeDependencyExclude -> '%(FileName).ni%(Extension)')" />
       <RunTimeDependencyExcludeFiles Include="@(RunTimeDependencyExclude -> '%(FileName).pdb')" />
+
       <AllResolvedRuntimeDependencies Include="@(RuntimeCopyLocalItems -> '%(FileName)%(Extension)');@(NativeCopyLocalItems -> '%(FileName)%(Extension)')">
         <File>%(Identity)</File>
       </AllResolvedRuntimeDependencies>
+      
       <RunTimeDependencyCopyLocalFile Include="@(AllResolvedRuntimeDependencies)"  Exclude="@(RunTimeDependencyExcludeFiles)"/>
       <RunTimeDependencyCopyLocal Include="@(RunTimeDependencyCopyLocalFile -> '%(File)')"  />
       <RunTimeDependencyCopyLocal Include="$(TargetingPackPath)/*" />
+      
+      <RunTimeArtifactsExcludeFiles Include="protononjit.dll" />
+      <RunTimeArtifactsExcludeFiles Include="linuxonjit.dll" />
+      
+      <RunTimeArtifactsIncludeFolders Include="/" />
+      <RunTimeArtifactsIncludeFolders Include="crossgen2/" />
+      <RunTimeArtifactsIncludeFolders Include="PDB/" />
+      <RunTimeArtifactsIncludeFolders Include="R2RTest/" />
+
+      <!-- Add binary dependencies to copy-local items -->
+      <RunTimeDependencyCopyLocal
+          Include="$(CoreCLRArtifactsPath)/%(RunTimeArtifactsIncludeFolders.Identity)*"
+          Exclude="@(RunTimeArtifactsExcludeFiles -> '$(CoreCLRArtifactsPath)/%(Identity)')">
+        <TargetDir>%(RunTimeArtifactsIncludeFolders.Identity)</TargetDir>
```

Comment:
You could simplify this by setting the metadata inline. I.e:

```xml
<RunTimeDependencyCopyLocal
          Include="$(CoreCLRArtifactsPath)/%(RunTimeArtifactsIncludeFolders.Identity)*"
          Exclude="@(RunTimeArtifactsExcludeFiles -> '$(CoreCLRArtifactsPath)/%(Identity)')"
          TargetDir="%(RuntimeArtifactsIncludeFolders.Identity)" />
```
```

Reply:
Thanks for teaching me MSBuild, I never knew that :D. Fixed in 2nd commit.